frontend/app/.server/domain/dtos/profile.dto.ts

```diff
@@ -44,3 +44,11 @@ export type UpdateAddressRequestDto = Readonly<{
     province?: string;
   };
 }>;
+
+export type ApplicantEligibilityDto = Readonly<{
+  clientId: string;
+  firstName: string;
+  lastName: string;
+  isEligible: boolean;
+  isEnrolled: boolean;
+}>;
```

Comment:
`isEligible` can only a value if `isEnrolled` value is `true`

```suggestion
export type ApplicantEligibilityDto = Readonly<{
  clientId: string;
  firstName: string;
  lastName: string;
}> &
  Readonly<
    | {
        isEnrolled: true;
        isEligible: boolean;
      }
    | {
        isEnrolled: false;
        isEligible?: undefined;
      }
  >;
```